header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -36,6 +36,8 @@ abstract class Abstract_Builder implements Builder {
 	const COLUMNS_NUMBER     = 'columns_number';
 	const COLUMNS_LAYOUT     = 'columns_layout';
 	const HEIGHT_SETTING     = 'height';
+	const BOTTOM_BORDER      = 'header_bottom_border';
+	const BORDER_COLOR       = 'header_border_color';
```

Comment:
Let's use simplified naming here. We already know these address the header rows. Let's use `border_width` and `border_color` 👀